recipes/Kili_fine_tuning_tutorial.ipynb

```diff
@@ -0,0 +1,2081 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.        exported_class = (nested_field[0]['name'])
</pre><p>I would just use exported_class = json_data["latestLabel"]["jsonResponse"]['CLASSIFICATION_JOB']['categories'][0]["name"]</p><p> </p><p>or use the label parsing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1259/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>